recipes/openssl/1.x.x/conanfile.py

```diff
@@ -85,7 +32,6 @@ class OpenSSLConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     options = {
         "no_threads": [True, False],
-        "no_zlib": [True, False],
```

Comment:
In the past, we did not remove options instantly, but printed a warning first. What is the current policy on that?

Reply:
this option is already removed for versions 1.1.1 in config_options